src/pages/invoices/common/helpers/resolve-column-width.ts

```diff
@@ -22,5 +22,5 @@ export function resolveColumnWidth(column: string) {
     discount: '10%',
   };
 
-  return mappings[property] || '';
+  return mappings[property] || '10%';
```

Comment:
Is this required? I generally feel that we shouldn't limit all columns, but rather let HTML drive the flow of spacing.

Reply:
Yeah, after testing it without limiting it to 10%, I feel like we can let HTML determine the spacing flow.